schemas/hpc-job-scheduler.md

```diff
@@ -7,22 +7,25 @@ Tools being targeted:
 - Slurm Job Logs
 - Flux Job Logs
 
-## Job Log Schema
+## Job Completion Log Schema
+
+This schema is for the job completion information of a completed job. In Slurm, this information is controlled by the `JobComp*` fields, such as `JobCompType` and `JobCompLoc` (see: <https://slurm.schedmd.com/slurm.conf.html> for more information).
 
 | **Field Name**         | **Field Explanation**                                                                | **Slurm**       | **Flux**         |
 | ---------------------- | ------------------------------------------------------------------------------------ | --------------- | ---------------- |
+| event.dataset\*        | Name of the dataset "slurm.joblog"                                                   | "slurm.joblog"  | "flux.joblog"    |
 | job.id                 | Unique identifier of the job                                                         | JobId           | id               |
-| user.name              | User name of who submitted the job                                                   | UserId          | username         |
-| user.id                | User ID number of who submitted the job                                              | UserId          | userId           |
-| group.id               | Group ID number of who submitted the job                                             | GroupId         |                  |
-| group.name             | Group ID number of who submitted the job                                             | GroupId         | jobspec.name     |
-| event.outcome          | The final state of the job                                                           | JobState        | result           |
+| user.name\*            | User name of who submitted the job                                                   | UserId          | username         |
+| user.id\*              | User ID number of who submitted the job                                              | UserId          | userId           |
+| group.id\*             | Group ID number of who submitted the job                                             | GroupId         |                  |
+| group.name\*           | Group ID number of who submitted the job                                             | GroupId         | jobspec.name     |
```

Comment:
Maybe this one is meant to be "ID name string" rather than "ID number"? Otherwise id just duplicated group.id.

Is jobspec.name really the right thing for group id?

Reply:
To your first point, yes the original text is "Group name of who submitted the job"